docs/source/python_api.rst

```diff
@@ -10,9 +10,6 @@ SMAUG Operators
    math
    tensor
 
-Building new SMAUG features
```

Comment:
I'd prefer keeping this separation since they are functionally quite different.

Reply:
Right. I think it makes sense to categorize these into `Python APIs` since these are also APIs used for building new features. I uploaded a screenshot for the new main page.